packages/aila/src/features/moderation/moderators/OpenAiModerator.ts

```diff
@@ -123,10 +123,7 @@ export class OpenAiModerator extends AilaModerator {
     );
 
     const log = aiLogger("aila:moderation:response");
-    log.info(
-      "Moderation response: ",
-      JSON.stringify(moderationResponse, null, 2),
-    );
+    log.info(JSON.stringify(moderationResponse));
```

Comment:
@mfdowland we've been logging the full response ... so i';; see if i can get old scores exported!

Reply:
Actually they were excluded in prod so looks like we won;t be getting these